spec/descriptions/agentMonitoringEvents.md

```diff
@@ -1,11 +1,5 @@
 This endpoint retrieves all agent monitoring events for the requested timeframe.
+The timeframe is defined as [from, to] or [to - windowSize, to].
 
-### Query Parameters:
-
-- **to:** The end of the requested timeframe as a Unix timestamp. The current service time is used as a default if not provided. Please note that it is recommended to fetch historical events with a delay using a `to` timestamp in the past due to eventual consistency. It can take up to **2 minutes** in some cases until an event update is reflected in the results.
-- **windowSize:** The size of the requested timeframe in milliseconds relative to `to`. If neither `windowSize` nor `from` is provided, then a default windowSize of *10 minutes* is used.
-- **from:** As an alternative to defining the `windowSize`, the `from` query parameter can be used, which spans the timeframe in range `[from, to)`.
-- **excludeTriggeredBefore:** Whether to exclude events that have been triggered before the requested timeframe in order to enable searching for events that have started within the given timeframe, excluding events that are previously active already. This is useful for 3rd party integrations that fetch events from Instana with a scheduled batch job in a fixed interval using tumbling windows, when you only care about new events.
-This option is more restrictive than `filterEventUpdates` and does not inform about event state updates that got `CLOSED` in the timeframe of the query if not also the start time of the event is within that query timeframe.
-- **filterEventUpdates:** Filters results to event updates only. This means that an event is only included when its event state changed in the given query timeframe. This is useful for 3rd party integrations that fetch events from Instana with a scheduled batch job in a fixed interval using a tumbling windows, when you care about event state updates.
-
+These are raw events coming directly from the sensors. 
```

Comment:
Honestly not sure whether they are initiated by a sensor, or by the agent. But effectively the Agent is sending them. And also because they are called "Agent Monitoring Issues" and not "Sensor Monitoring Issues", I guess it might be better to say the following instead:
> These are raw events coming directly from the **Instana agent.**

Lastly, the wording might be a bit vague. "are coming directly from the sensors" sounds a bit like this API is pulling the data ad-hoc from the agent/sensor. And I think the interesting part is in the line below. So maybe just remove this line? Or combine it to a single sentence:
> These events provide information about what is hindering the Instana agent from providing better monitoring, such as in case of lack of permissions or missing configuration.

Reply:
Updated, with very slightly different wording near the end